command/device/create.go

```diff
@@ -65,19 +69,87 @@ func Create(ctx context.Context, params *CreateParams, cred *config.Credentials)
 		)
 	}
 
-	iotClient, err := iot.NewClient(cred)
+	iotApiRawClient := iotapiraw.NewClient(cred)
```

Comment:
Why do you need another iot api client? Can't we use the existing one (or regenerate it if is lacking of required endpoints)? If I'm not wrong the last generated client is quite recent

Reply:
The arduino `iot-client-go` doesn't expose two of the endpoints needed, I think because they are not "business oriented" and they are more for technical purposes.

The missing endpoint:
- `/iot/v2/binaries` 
- `/iot/v1/supported/devices`

I don't know if these endpoints are useful in the go client